k2qc/core.py

```diff
@@ -120,7 +127,12 @@ def verify_positive_flux(self):
             warnings.simplefilter("ignore")  # ignore NaN comparison warnings
             mask = self.tpf[1].data['QUALITY'] == 0
             flux_plus_bkg = self.tpf[1].data['FLUX'][mask] + self.tpf[1].data['FLUX_BKG'][mask]
-            assert (flux_plus_bkg < 0).sum() == 0
+            if (flux_plus_bkg < 0).sum() > 0:
+                badpixels = np.argwhere(flux_plus_bkg < 0)
+                for pix in badpixels:
+                    print("Negative flux values found at frame {},"
```

Comment:
Ok, done! I also fixed the issue regarding the frame number (before it was printing the frame number relative to ``quality==0`` frames; now it prints the actual frame number).